flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/utils/FlinkRelOptUtil.scala

```diff
@@ -90,6 +90,58 @@ object FlinkRelOptUtil {
     sw.toString
   }
 
+  /**
+   * Converts a sequence of relational expressions to a string. This is different from
+   * [[RelOptUtil]]#toString and overloaded [[FlinkRelOptUtil]]#toString on following points:
+   *   - Generated string by this method is in a tree style
+   *   - Generated string by this method may have more information about RelNode, such as
```

Comment:
nit: -> 'Generated string by this method may have available [[PlanAdvice]]'? 
'retractionTraits' is not easy to understand for developers and I didn't see more 'withXX' args enabled  by default.